docs/developer-guide/release.md

```diff
@@ -22,55 +22,86 @@ Add an entry in the changelog like this:
-    - [ ] use the label to filter the issues on github and assign to all the issue the milestone created
-    - [ ] remove the temporary label "current release"
-- [ ] Update `CHANGELOG.md`. [Instructions](https://mapstore.readthedocs.io/en/latest/developer-guide/release/#changelog-generation) - both master and stable
+    - [ ] assign the label "current-release" to all the issues and Prs of the current zenhub release
+    - [ ] use the label to filter the issues on github and assign to all the issues and Prs the milestone created
+    - [ ] remove assignments of "current-release" 
+- [ ] Prepare PR for updating `CHANGELOG.md` for **master** and **stable** [Instructions](https://mapstore.readthedocs.io/en/latest/developer-guide/release/#changelog-generation)
 - [ ] Fix `pom.xml` dependencies stable versions ( no `-SNAPSHOT` usage release).
 - [ ] Update the version of java modules on the stable branch to a stable, incremental version.
-    - [ ] Run `mvn versions:set -DnewVersion=<VERSION> -DprocessAllModules -DgenerateBackupPoms=false` to update package version, where <VERSION> is the version of the java packages (e.g. `1.2.2`).
+    - [ ] Run `mvn release:update-versions -DdevelopmentVersion=<version> -Pprinting,printingbundle,release
+` to update package version, where <VERSION> is the version of the java packages (e.g. `1.2.2`).
     - [ ] Manually update project pom templates to use `mapstore-services` of `<VERSION>`
-- [ ] Release a stable `mapstore-services`. (from `2022.01.xx` also mapstore-webapp should be deployed for new project system and product).
+- [ ] Release a stable `mapstore-services`. (from `2022.01.xx` also mapstore-webapp (java/web) should be deployed for new project system and product). 
+  - [ ] for mapstore-webapp use `mvn clean install deploy -f java/pom.xml`
+  - [ ] for the product use `mvn clean install deploy -f product/pom.xml` if this does not work it can be ignored
 - [ ] create on [ReadTheDocs](https://readthedocs.org/projects/mapstore/) project the version build for `YYYY.XX.xx` (click on "Versions" and activate the version of the branch)
 - [ ] Test on QA [https://qa-mapstore.geosolutionsgroup.com/mapstore/](https://qa-mapstore.geosolutionsgroup.com/mapstore/)
-    * Any fix must be done on **YYYY.XX.mm**. The fixes will be manually merged on master
+    * Any fix must be done on **YYYY.XX.xx**. The fixes will be manually merged on master
     * Test **everything**, not only the new features
-- [ ] Test [Binary](https://build.geo-solutions.it/jenkins/view/MapStore2/job/MapStore2-QA-Build/ws/release/target/) (take the mapstore2-QA-<RELEASE_BRANCH>-bin.zip)
-- [ ] Lunch the [stable deploy](https://build.geo-solutions.it/jenkins/view/MapStore2/job/MapStore2-Stable/) to install the latest stable version on official demo
+    * Test creation  of a standard project and run the internal backend, with `npm run backend` and `npm start`, then check localhost:8081 that homepage is empty
```

Comment:
We have also to configure webpack now to do this test, so I should not specify too much the instructions. Hopefully it will become the new standard way of dev in the next major, so we can remove it in the near future :+1: 

```suggestion
    * Test creation of a standard project 
    * Test the local dev backend (`npm run backend`)
```

Reply:
this line was specifically targeting the test of a standard project that was using internal backend, 
so I would leave this at least

consider that giving less details is also problematic
and having this automated in an hypotetical future CI workflow 
does not leave out the fact the we have to do this kind of testing